### PR TITLE
added new route /:slug/:tier.(svg|png)

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: server/controllers/banner.js [hunk 1]
```diff
@@ -129,6 +131,89 @@ module.exports = {
...
+          res.setHeader('content-type','image/png');
+          return utils.svg2png(svg)
+      }
+    })
+    .then(svg => {
+      res.setHeader('Cache-Control', 'public, max-age=300');
+      res.send(svg);
+    })
+    .catch(e => {
+      console.log("An error happened");
```
Comment (asood123) [inline server/controllers/banner.js]: no need for this console.log

**Comment by asood123** [comment]
Love this PR!

**Comment by mxstbr** [comment]
Yessssss!!

File: package.json [hunk 2]
```diff
@@ -101,6 +101,7 @@
     "robots.txt": "1.1.0",
     "serialize-javascript": "1.2.0",
     "serve-favicon": "2.3.0",
+    "svg-to-png": "^3.1.0",
```
Comment (sedubois) [inline package.json]: freeze dependency version?

File: package.json [hunk 3]
```diff
@@ -113,6 +114,7 @@
     "eslint-config-opencollective": "1.1.0",
     "eslint-plugin-react": "3.16.1",
     "expect": "1.18.0",
+    "foreman": "^1.4.1",
```
Comment (sedubois) [inline package.json]: freeze dependency version?

**Pull request merged**